plutus-conformance/executables/test-utils/Spec.hs

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This module is using two space indentation. Now that you've renamed this module, I'd just fourmolu it.